### PR TITLE
Bug/jel 365 update manage iq worker

### PR DESCRIPTION
**Pull request by OBCENEIKON**

### CODE REVIEW

**Comment by OBCENEIKON** [comment]
Closes #365

File: app/workers/manage_iq.rb [hunk 1]
```diff
@@ -75,12 +75,15 @@ def payload
   end
 
   def request
-    # TODO: verify_ssl needs to be changed, this is the only way I could get it to work in development.
+    # On OS/X, for development only, it may be easiest just to disable certificate verification because the certificates are stored in the keychain, not the file system
+    # https://github.com/plataformatec/devise/wiki/OmniAuth:-Overview
+    ssl_verify = Rails.env.development? ? OpenSSL::SSL::VERIFY_NONE : OpenSSL::SSL::VERIFY_PEER
```
Comment (chriskacerguis) [inline app/workers/manage_iq.rb]: We wouldn't want to do this in production, as that is a security issue.  Let's make sure we only do this in env = "development"

Comment (chriskacerguis) [inline app/workers/manage_iq.rb]: @OBCENEIKON ???

Comment (calebthompson) [inline app/workers/manage_iq.rb]: That's what this line does.

**Comment by calebthompson** [comment]
LGTM

File: app/workers/manage_iq.rb (hunk 1 above)
Comment (chriskacerguis) [inline app/workers/manage_iq.rb]: /facepalm

**Pull request merged**